test/unit/app/models/landing_page/base_block_test.rb

```diff
@@ -11,4 +11,10 @@ class BaseBlockTest < ActiveSupport::TestCase
     assert subject.invalid?
     assert_equal ["Type can't be blank"], subject.errors.to_a
   end
+
+  test "raises error when presenting an invalid block to publishing api" do
+    subject = LandingPage::BaseBlock.new({}, [])
+    assert subject.invalid?
+    assert_raises(StandardError, match: /Type can't be blank/) { subject.present_for_publishing_api }
```

Comment:
Should we also assert that `cannot present invalid block to publishing api` is included in the StandardError?
Not immediately obvious the corresponding line has test coverage otherwise.